### DISCUSSION TITLE
Next release date?

### DISCUSSION THREAD

**Discussion by GoldsteinE** [go-acme/lego#1827] [Q&A] [answered]
Hi!

The last released version of lego contains a bug in LuaDNS provider that was fixed in master. I’m currently building lego from master to work around it, but I’m wondering about release schedule of this project. Is there a release planned?

**Comment by ldez** [comment, marked as answer]
Hi,

We don't have a schedule, but I think we will create a release in 1 or 2 weeks maybe less.
The time to review and merge some PRs.

**Reply by GoldsteinE** [reply]
Thanks for the answer!

**Reply by ldez** [reply]
the new release is available https://github.com/go-acme/lego/releases/tag/v4.10.0
